index.html

```diff
@@ -50,6 +50,17 @@ <h5 class="card-title">developブランチ（masterブランチではないブ
           </div>
     </div>
 
+    <div class="container">
+        <div class="card">
+            <h5 class="card-header">せお丸さんの【Git入門】4</h5>
+            <div class="card-body">
+              <h5 class="card-title">また別のブランチを作って、開発する時</h5>
```

Comment:
```suggestion
              <h5 class="card-title">また別のブランチを作って、開発する時</h5>
```
追記でなぜもう一つ新たに作る必要があるのかを書きましょう